osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -50,6 +50,12 @@ public abstract class SongSelect : ScreenWithBeatmapBackground, IKeyBindingHandl
 
         public FilterControl FilterControl { get; private set; }
 
+        /// <summary>
+        /// Whether this song select instance should take control of the global track,
+        /// applying looping and preview offsets.
+        /// </summary>
+        protected virtual bool ControlGlobalMusic => true;
```

Comment:
I think this PR is missing a file? It's not overridden anywhere as far as I can see.